src/drake/shell.clj

```diff
@@ -96,7 +102,7 @@
         proc (.exec (Runtime/getRuntime)
```

Comment:
I don't think printing a foo.txt matters.

The ```[clojure]``` protocol isn't implemented, so all I care about is [c4rows], which doesn't do output on its own. The Drake step output is all that matters, and that is handled Drake-side instead of within the step, so that's irrelevant to this feature.

If require isn't supported in c4rows, this is one less thing I have to worry about supporting. If require is supported, I'd like to use it so I can figure out how to require Clojure files in the same project as the included Drakefile.